### PR TITLE
Add complete help text for cluster config

### LINKED ISSUE #354: Generate help text in a jinja template
Currently the help text displayed via `lib/common.sh` is based on the old config and will only display how to use kubectl for the old cluster name.

Instead, create the help text from a jinja template with ansible and just cat that file in common.sh.

### PR DESCRIPTION
**Pull request by joejulian**
Fixes #354

### CODE REVIEW

**Comment by joejulian** [comment]
Sample output:

```
PLAY RECAP ***********************************************************************************************
localhost                  : ok=112  changed=0    unreachable=0    failed=0   

To use kubectl:
---------------

    kubectl --kubeconfig=/home/jjulian/.kraken/jjmaster/admin.kubeconfig <kubectl command>

To use helm:
------------

    KUBECONFIG=/home/jjulian/.kraken/jjmaster/admin.kubeconfig helm \
      --home /home/jjulian/.kraken/jjmaster/.helm <helm command>

To ssh:

    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config etcd-1
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config etcd-2
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config etcd-3
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config etcd-4
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config etcdEvents-1
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config etcdEvents-2
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config etcdEvents-3
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config etcdEvents-4
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config master-1
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config master-2
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config clusterNodes-1
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config clusterNodes-2
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config specialNodes-1
```

File: ansible/roles/kraken.cluster_common/templates/help.txt.jinja2 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    KUBECONFIG={{ config_base }}/{{ cluster.name }}/admin.kubeconfig helm \
+      HELM_HOME={{ config_base }}/{{ cluster.name }}/.helm \
+      helm <helm command>
+{% endfor %}
+
+To ssh:
+
+{% for cluster in kraken_config.clusters %}
+{%   for node in cluster.nodePools       %}
+{%     for number in range(1,node.count) %}
```
Comment (davidewatson) [inline ansible/roles/kraken.cluster_common/templates/help.txt.jinja2]: Does it make sense to limit how many examples we give for a given nodePool? For example:

```
To ssh:

    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config clusterNodes-1
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config clusterNodes-2
...
    ssh -F /home/jjulian/.kraken/jjmaster/ssh_config clusterNodes-4
```

Pro: More user friendly
Con: Easier to parse from a script

Comment (joejulian) [inline ansible/roles/kraken.cluster_common/templates/help.txt.jinja2]: Taking the stance that users should be running k2cli, this is for developers so we can just copy-paste.

**Comment by coffeepac** [comment]
retest this please

**Comment by joejulian** [comment]
test this please

**Comment by joejulian** [comment]
test this please

**Pull request merged**